src/components/CharacterList/CharacterList.tsx

```diff
@@ -8,14 +8,12 @@ import { Container, CurrentPageText, PaginationBox } from "./styles";
 
 export function CharacterList() {
   const [page, setPage] = useState(1);
-  const { data } = useQuery(
-    ["characters", page],
-    () => api.getCharacters(page),
-    {
-      keepPreviousData: true,
-      staleTime: 5000,
-    }
-  );
+  const { data } = useQuery({
+    queryKey: ["characters", page],
+    queryFn: () => api.getCharacters(page),
+    placeholderData: keepPreviousData,
```

Comment:
Can you tell me why you replaced the `keepPreviousData` option with `placeholderData`? 
What exactly does the imported `keepPreviousData` item do?